### PR TITLE
fix: Add PR to projects now trigger all branches

### PR DESCRIPTION
**Pull request by manueleckarth**

### CODE REVIEW

**Review by holgerson97 — APPROVED** [review]

File: .github/workflows/projects_add_pr_to_projects.yaml [hunk 1]
```diff
@@ -2,8 +2,6 @@ name: Add PR to Project
 on:
   pull_request:
     types: [opened]
-    branches:
-      - main
 
 jobs:
   add-to-project:
```
Comment (holgerson97) [inline .github/workflows/projects_add_pr_to_projects.yaml:7]: ey

**Review by holgerson97 — CHANGES_REQUESTED** [review]

File: .github/workflows/projects_add_pr_to_projects.yaml (hunk 1 above)
Comment (holgerson97) [reply .github/workflows/projects_add_pr_to_projects.yaml:7]: snake_case

Comment (manueleckarth) [reply .github/workflows/projects_add_pr_to_projects.yaml]: Sorry, Schlaganfall

**Review by holgerson97 — APPROVED** [review]
LGTM

**Pull request merged**